CoreScriptsRoot/ServerCoreScripts/ClientChat/ChannelsBar.lua

```diff
@@ -281,6 +283,12 @@ function methods:CreateTweeners()
 	
 end
 
+function methods:WaitUntilParentedCorrectly()
```

Comment:
why is this necessary? I'm not a huge fan of this method having to be repeated for every file. Are the sizes of the guis wrong when the script is first run? Do we do something similar in the current chat script?